build/makelib/golang.mk

```diff
@@ -156,6 +156,14 @@ go.test: go.vet go.fmt $(CGO_PREREQS)
 #	@$(GOHOST) test -v -i $(GO_PKG_FLAGS) $(GO_NONSTATIC_FLAGS) $(GO_ALL_PACKAGES)
 	@$(GOHOST) test -cover $(GO_PKG_FLAGS) $(GO_NONSTATIC_FLAGS) $(GO_ALL_PACKAGES)
 
+.PHONY: go.lint
+go.lint:
```

Comment:
is there any overlap with the existing `vet` target?  Does that supercede the need for this new `lint` target?

The `vet` target should probably be added to the Jenkinsfile too either way.